openml-h2o/src/main/java/com/feedzai/openml/h2o/server/H2OApp.java

```diff
@@ -168,6 +184,9 @@ public Path export(final Model model,
                        final Path exportDir) throws IOException, ModelTrainingException {
         logger.info("Exporting model {} to {}", model._output._job._result.toString(), exportDir.toAbsolutePath().toString());
         if (model.haveMojo()) {
+            if (model instanceof IsolationForestModel && ((IsolationForestModel) model)._output._min_path_length == Long.MAX_VALUE) {
+                ((IsolationForestModel) model)._output._min_path_length = Integer.MAX_VALUE;
```

Comment:
Add implementation comment here (even if brief).

Reply:
Please do check is my brief explanation is perceivable.

Reply:
Hint that it is a bug in H2O code that we're working around with this hack.